features/include/pcl/features/impl/rops_estimation.hpp

```diff
@@ -493,7 +487,7 @@ pcl::ROPSEstimation <PointInT, PointOutT>::getDistributionMatrix (const unsigned
     matrix (row, col) += 1.0f;
   }
 
-  matrix /= static_cast <float> (number_of_points);
+  matrix /= static_cast <float> (std::min<std::size_t> (1, cloud.points.size ()));
```

Comment:
`std::max`, not `min`. Also, maybe use eigen

Reply:
Will this work (to avoid explicit casting)?

```cpp
matrix /= std::max<float>(1, cloud.points.size());
```

Reply:
Yes, that'll work